spec/models/claim/advocate_hardship_claim_spec.rb

```diff
@@ -78,6 +80,7 @@
   let(:claim) { build(:advocate_hardship_claim) }
 
   it_behaves_like 'a base claim'
+  it_behaves_like 'a claim not eligible for unused materials fee'
```

Comment:
from a business perspective is this true. hardship claims are similar to final claim in most respects. can you check with the AB

Reply:
We should probably have the warning on any claim where unused materials is eligible. A hardship claims is eligible for example:

![Screenshot 2021-06-16 at 15 03 15](https://user-images.githubusercontent.com/7016425/122233882-5165e700-ceb4-11eb-8f49-f4c161b9c7e4.png)